pose-detection/src/blazepose/detector.ts

```diff
@@ -134,6 +136,7 @@ export class BlazeposeDetector extends BasePoseDetector {
     const config = validateEstimationConfig(estimationConfig);
 
     if (image == null) {
+      this.regionOfInterest = null;
```

Comment:
It looks strange that `regionOfInterest` is a state of the detector. Shouldn't it be just parameter or the call?